protocols/bgp/server/fsm.go

```diff
@@ -70,6 +70,8 @@ type FSM struct {
 	reason     string
 	active     bool
 
+	counters *fsmConters
```

Comment:
There is no need to have have pointer here. The struct should be embedded directly.